src/bundle/ui-dev/src/modules/universal-discovery/components/grid-view/grid.view.js

```diff
@@ -53,11 +53,9 @@ const GridView = ({ itemsPerPage }) => {
     }, [loadedLocations, dispatchLoadedLocationsAction, isLoading]);
 
     return (
-        <div className="c-grid">
+        <div className="c-grid" onScroll={loadMore}>
```

Comment:
I think it would be good to scroll only items and breadcrumbs should stay always visible.